src/sass/partials/_header.scss

```diff
@@ -178,11 +178,12 @@
   transition-duration: 0.3s;
   transition-timing-function: cubic-bezier(0.4, 0, 0.2, 1);
 
-  &:hover {
-    border-color: getColor(light-orange);
-    color: getColor(light-orange);
-    scale: 1.1;
-    cursor: pointer;
+
+  &:hover,
+  &:focus {
+    background-color: #ff6b01;
+    border-color: #ff6b01;
+
```

Comment:
jeśli w pozostałych miejscach dodajesz scale a w tym usuwasz to gdzie tu logika? szkoga że nie użyłeś getColor